app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Layout name and the java class names should match, we cannot have the activity called news activity and the layout called activity main

Reply:
done